plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFlags.java

```diff
@@ -122,7 +132,12 @@ public void adjust(Mechanism mechanism) {
         // @input ObjectTag
         // @description
         // Modifies a flag on this item, using syntax similar to <@link command flag>.
-        // For example, 'flag:myflagname:!' will remove flag 'myflagname', or 'flag:myflagname:3' sets flag 'myflagname' to value '3'.
+        // @example
+        // # Removes flag 'myflagname' from your held item
+        // - flag <player.item_in_hand> myflagname:!
+        // @example
+        // # Sets flag 'myflagname' on your held item to value '3'
+        // - flag <player.item_in_hand> myflagname:3

```

Comment:
Don't think these are correct? this is the `flag` ItemTag mechanism, not the `flag` command